components/runtime/vala/vala.p5m

```diff
@@ -19,21 +19,22 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2013 David Hoeppner. All rights reserved.
+# Copyright (c) 2017 Andreas Wacknitz
 #
 
 # pull the manpages out of the component dir
 <transform file path=usr/share/man/(man3/.+$) -> set action.hash %<\1> >
-
 <transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
```

Comment:
Ok, seems you are right. I've found
defaults:<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
in the transformations folder.